shipstation/api.py

```diff
@@ -20,6 +20,7 @@ def as_dict(self):
             if value is None:
                 d[key] = None
             else:
+                if type(value) == unicode: value = value.encode('ascii', errors='replace')                
```

Comment:
Actually, now that I've thought about it, this would be a good case to add a test to the tox suite as well.